spec/datadog/tracing/contrib/opensearch/quantize_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'datadog/tracing/contrib/support/spec_helper'
+require 'opensearch'
+
+require 'ddtrace'
+require 'datadog/tracing/contrib/opensearch/quantize'
+
+RSpec.describe Datadog::Tracing::Contrib::OpenSearch::Quantize do
```

Comment:
If we copy over `quantize` code from ElasticSearch, then it makes sense to keep this spec to check its behavior. Otherwise, if we promote this to `Contrib`, this spec should live there instead.

Reply:
Code was copied over from `quantize` in Elasticsearch